util/rand_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package util
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestRandString(t *testing.T) {
+	s, err := RandomID("prefix.")
+	t.Logf("Rand string: %s", s)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !strings.HasPrefix(s, "prefix.") {
+		t.Errorf("Invalid prefix: %s", s)
+	}
+	if len(s)-len("prefix.") != 52 {
```

Comment:
Good catch, yeah the padding remove should only happen from the returned base32 string. Fixed with test.